migrations/tenant/0012-notes-settings@rename-table-to-note-settings.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE IF EXISTS public.notes_settings 
+    RENAME TO note_settings
```

Comment:
Tests still fail to run. Probably you need to add check if `note_settings` table already exists and do not rename `notes_settings` then